frontend/src/app/modules/common/autocomplete/user-autocompleter.component.ts

```diff
@@ -34,9 +34,10 @@ import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {Observable} from "rxjs";
 import {map} from "rxjs/operators";
 import {DebouncedRequestSwitchmap, errorNotificationHandler} from "core-app/helpers/rxjs/debounced-input-switchmap";
-import {WorkPackageNotificationService} from "core-components/wp-edit/wp-notification.service";
+import {HalResourceNotificationService} from "core-app/modules/hal/services/hal-resource-notification.service";
 import {NgSelectComponent} from "@ng-select/ng-select";
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
+import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/e1c1b00c5a6c973508c7538a865d197276aa638c/frontend/src/app/modules/common/autocomplete/user-autocompleter.component.ts#L41"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>